azure/durable_functions/models/OrchestratorState.py

```diff
@@ -1,5 +1,5 @@
 import json
-from typing import List, Any, Dict, Optional, Union
+from typing import List, Any, Dict, Optional
```

Comment:
When running the pipeline, there is an error about this that the Union hasn't been used. Thus, I deleted it. Not sure why this didn't get triggered before.

Reply:
Not sure either, though our types need some serious updating. In any case, if yhis unblocks us, that's good.